### ISSUE TITLE
Tweak 'Foundation Opportunity' dashboard in Salesforce

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/Engagement#5]
The [Foundation Opportunity dashboard](https://na12.salesforce.com/01ZU0000000UQWG) in Salesforce needs a bit of attention. 

Driver: @edrushka
Decision-maker: @benrito 
Consulted: Lynn
Informed: @OpenMatt 

@benrito please outline what you would like done

**Comment by davidascher** [comment]
This doesn't feel like a dev-needing issue.  Who did the first version?

**Comment by edrushka** [comment]
I do not know. Perhaps it was @secretrobotron?

**Comment by benrito** [comment]
Currently in a meeting but will comment on this shortly

**Comment by benrito** [comment]
We have designed a semi-public dashboard that will be generated based off the Salesforce database.

Building this will require some front-end engineering on top of Salesforce APIs.

Intuitively it shouldn't be too hard, but it will require someone to dive into VisualForce for a day or two: https://developer.salesforce.com/page/An_Introduction_to_Visualforce
![pipelinemockup](https://cloud.githubusercontent.com/assets/541325/6794208/2a553eb2-d18c-11e4-88a3-763dbc484d9c.jpg)

**Comment by benrito** [comment]
This is what we're able to generate without doing any custom work with VisualForce:

![pipelinelive](https://cloud.githubusercontent.com/assets/541325/6794226/520ff35c-d18c-11e4-90ed-7294efd89b41.png)

**Comment by edrushka** [comment]
@simonwex is this something Scott or Ali can do? If not, I'll need a dev from another "team" to tackle this.

**Comment by simonwex** [comment]
No one on the team has any experience here, but this will be in Scott's wheelhouse.

**Comment by edrushka** [comment]
@benrito can you please help get @alicoding set up with a dev account and access to the sandbox instance so he can begin to tackle this? ty

**Comment by benrito** [comment]
Holding on whether Lynn believes this is still a priority

**Comment by edrushka** [comment]
@benrito Did you connect with Lynn? Still want to do this?

**Comment by adamlofting** [comment]
I've created Salesforce accounts for @ScottDowne and @alicoding now.

I did some proof of concept work on something very similar last year when we started migrating the engagement pipeline data from Google Docs to Salesforce for the [Gauntlet Dashboard](http://gauntletdash.herokuapp.com/).

@simonwex @ScottDowne @alicoding you'll probably be happy to see [JSForce](https://jsforce.github.io/) which allows you to work in node/js land for some of this. My POC looked up an existing report in salesforce and extracted the data. This might remove the need to learn VisualForce.
